src/jmh/java/io/deephaven/csv/benchmark/util/DateTimeToLongParser.java

```diff
@@ -18,7 +20,9 @@ public long parse(final String s) {
         }
     }
 
-    class Deephaven implements DateTimeToLongParser {
+    enum Deephaven implements DateTimeToLongParser {
+        INSTANCE;
```

Comment:
Unfortunately we cannot make this class a singleton because it's mutable (therefore not threadsafe). It will randomly and mysteriously break code that happens to be doing concurrent date time parsing.

Reply:
Oops. Updated, added some more docs, and new constructor.